core-it-suite/src/test/java/org/apache/maven/it/MavenITmng4412OfflineModeInPluginTest.java

```diff
@@ -93,10 +93,10 @@ public void testitCollector()
         verifier.deleteDirectory( "target" );
         verifier.deleteArtifacts( "org.apache.maven.its.mng4412" );
         verifier.filterFile( "settings-template.xml", "settings.xml", "UTF-8", verifier.newDefaultFilterProperties() );
-        verifier.addCliOption( "-Pcollector" );
-        verifier.addCliOption( "--offline" );
-        verifier.addCliOption( "-s" );
-        verifier.addCliOption( "settings.xml" );
+        verifier.addCliArgument( "-Pcollector" );
+        verifier.addCliArgument( "--offline" );
+        verifier.addCliArgument( "-s" );
+        verifier.addCliArgument( "settings.xml" );
```

Comment:
These successive calls to `addCliArgument` could be possibly replaced by single call to
```
addCliArguments("-Pcollector", "--offline", "-s", "settings.xml");
```
I am not sure though if that's making things easier or harder to read (it may be just matter of preference). In any case, those would be done in different PR anyway.